packages/core/src/schema/models/integrations.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  bigint,
+  bigserial,
+  index,
+  text,
+  timestamp,
+  unique,
+  varchar,
+  jsonb,
+} from 'drizzle-orm/pg-core'
+
+import { latitudeSchema } from '../db-schema'
+import { timestamps } from '../schemaHelpers'
+import { users } from './users'
+import { workspaces } from './workspaces'
+import { IntegrationTypes } from '@latitude-data/constants'
+import { CustomMCPConfiguration } from '../../services/integrations/helpers/schema'
+
+export const integrationTypesEnum = latitudeSchema.enum('integration_types', [
+  IntegrationTypes.CustomMCP,
+])
+
+export type IntegrationProviderConfig<P extends IntegrationTypes> =
+  P extends IntegrationTypes.CustomMCP ? CustomMCPConfiguration : never
+
+export const integrations = latitudeSchema.table(
+  'integrations',
+  {
+    id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
+    name: varchar('name').notNull(),
+    type: integrationTypesEnum('type').notNull(),
```

Comment:
You can type this. Also call the column `integration_type`. I try to avoid ultra generic column names because then you can have conflicts on joins for example